x-pack/plugins/apm/server/lib/service_map/get_service_map.ts

```diff
@@ -137,9 +138,42 @@ async function getServicesData(options: IEnvOptions) {
   );
 }
 
+async function getAnomaliesData(options: IEnvOptions) {
+  const { setup } = options;
+  const { client } = setup;
+  const params = {
+    index: '.ml-anomalies-*',
```

Comment:
`.ml-anomalies-*-high_mean_response_time`
Perhaps use the existing helper `getMlIndex('*')`

Reply:
Updated in 3bbd7dd.